airtime_mvc/application/views/scripts/login/password-restore.phtml

```diff
@@ -1,10 +1,11 @@
 <div class="login_box">
   <div class="logobox">&nbsp;</div>
   <h2><?php echo _("Reset password") ?></h2>
-  
+
   <div id="login" class="login-content clearfix">
     <p class="light">
-        <?php echo _("Please enter your account e-mail address. You will receive a link to create a new password via e-mail.")?>
+        <?php echo _("Enter your account e-mail address or your username (or both, if you have multiple accounts
+                      using the same email address), and we'll send you a link to reset your password.")?>
```

Comment:
Also, we really need to force each user to have a unique email address. The way this works works is super confusing. Maybe we can tackle this when we do invitation emails.